board/models.py

```diff
@@ -2,10 +2,47 @@
 
 from django.contrib.auth.models import User
 from django.db import models
+from django.utils import timezone
 
 # Create your models here.
 
 
+class Image(models.Model):
+    """
+    A database model representing an image, containing a name and the BLOB of the actual image.
+
+    This is a separate table to make indexing the other tables more efficient, and make extraction
+    of one image simple and convenient. Each image in a Board or Post has a one-to-one relationship
+    with an image stored in this table.
+
+    Class Attributes
+        name -> `CharField`: A charfield with max length 100 with the name of the image.
+        photo -> `BinaryField`: A field with the stored image BLOB.
+        uuid -> `UUIDField`: A unique, non-editable uuid4 UUID for each image, used to locate it.
+    """
+
+    name = models.CharField(max_length=100)
+    photo = models.BinaryField()
+    uuid = models.UUIDField(default=uuid.uuid4, unique=True, editable=False)
```

Comment:
Can you also add a `created_at` field for the images (one day it'll probably be useful).

Reply:
:ok_hand: